.storybook/themes/commerce.scss

```diff
@@ -390,10 +445,36 @@
 				quilted-image__wrapper: (
 					grid-template-areas: "top top" "bottom bottom",
 				),
+				story-carousel: (
+					components: (
+						carousel: (
+							--slides: 4,
+						),
+						carousel-actions: (
+							display: flex,
+							margin-top: -7rem,
+						),
+						carousel-track: (
+							justify-self: center,
+							max-width: 88%,
+						),
```

Comment:
Also in desktop/components, except the margin shift.  Why are we applying negative margin (and so much of it)?...

Reply:
To move the buttons up more to be centered with the images and not the whole card itself.